portfolio/src/main/webapp/index.html

```diff
@@ -37,8 +39,9 @@
                          surrounds. Friendship is very important to me. I value the connections I have made with 
                          people on my educational journey as they increase my level of happiness, provide support 
                          during stressful situations, improve my self-worth and confidence. In the words of Winnie 
-                         the Pooh, "a friend is one of the nicest things you can have and one of the best things 
-                         you can be". </p>
+                         the Pooh, 
+                         "a friend is one of the nicest things you can have and one of the best things you can be". 
+                        </p1>
```

Comment:
I've never heard of p1. Wondering what's the difference between this and <p> tag?

Reply:
I created p1 in my style.css to specify font differences